test/ui-automation/test/helpers/wallet.js

```diff
@@ -213,6 +222,23 @@ async function _changeLocale() {
     });
 }
 
+async function _changeVaultLocale() {
+    const localeSwitcherLink = await $("a*=Français");
+    await localeSwitcherLink.waitForExist();
+    await localeSwitcherLink.click();
+    // Verifying lang attribute is set to French on the page
+    await expect(browser).toHaveUrlContaining('https://wallet.trustbloc.local:8091/fr/');
```

Comment:
It's better to get the base url from global browser var than hardcoding this.

Reply:
Also, just wondering if these validations can be written in spec file rather than this helper file - its better to only keep core/common code in this file

Reply:
Make sense i will update